content/spin/v2/rust-components.md

```diff
@@ -334,6 +335,10 @@ Handlers within a `Router` can be sync or async. Use `Router`'s "plain" methods
 
 > For further reference, see the [Spin SDK HTTP router](https://fermyon.github.io/rust-docs/spin/main/spin_sdk/http/struct.Router.html).
 
+## Granular Routing
+
+Granular routing allows for more precise control over the request-handling process. Based on the [routefinder](https://crates.io/crates/routefinder) crate, granular routing in Spin allows users to define dynamic routes where certain parts of the URL can change based on the input provided. A colon (`:`) precedes a variable segment in a URL. This segment captures part of the URL as a variable, which can be used in your route handler. For example, `/goodbye/:planet`. In this route, `:planet` is a variable segment. When a user accesses a URL like `/goodbye/mars`, the router matches this to the `/goodbye/:planet` pattern, and `mars` is captured as the value of `planet`. The `RouteMatch` for this request would include the handler associated with the pattern and a parameter map containing `planet: mars`; this parameter map is what your application uses to handle and subsequently process the request.
```

Comment:
RouteMatch is an implementation detail of the trigger: it's not visible to guests.  Guests still get the standard HTTP API, with matches delivered by headers in a similar way to current `spin-path-info`.